runtime/src/accounts_db.rs

```diff
@@ -76,7 +79,7 @@ pub struct AccountInfo {
 }
 /// An offset into the AccountsDB::storage vector
 pub type AppendVecId = usize;
-pub type InstructionAccounts = Vec<Account>;
+pub type InstructionAccounts = Vec<(Fork, Account)>;
```

Comment:
when I tried changing this type, I ran into problems with process_instruction(), which really wants a slice of accounts...  Instead, I've added yet another vector to the return signature of load(), a la credits

Reply:
Yea. I tried to get that working without affecting those interfaces but maybe you're right another argument is the right way. I ended up just cloning them at the moment which isn't great.